devtools_m2/cypress/integration/Wishlist/steps.js

```diff
@@ -0,0 +1,111 @@
+import { Given, When, Then } from "cypress-cucumber-preprocessor/steps"
+import { mockServerClient } from "mockserver-client"
+
+const Mockclient = mockServerClient("localhost", 1080);
+
+When('I create an account', function () {
+    cy.contains('Create an Account').click()
+    cy.get('#form-validate').within(function () {
+        cy.get('input[name="firstname"]').type('Test')
+        cy.get('input[name="lastname"]').type('User')
+        cy.get('input[name="email"]').type('testuser@example.com')
+        cy.get('input[name="password"]').type('blahblah123!!!')
+        cy.get('input[name="password_confirmation"]').type('blahblah123!!!')
+        cy.contains('Create an Account').click()
+    })
+})
+
+When('I add a {string} widget to my wishlist', function (type) {
+    cy.visit(`/widget-1.html`)
+    switch (type) {
+        case 'configurable':
+          cy.get('#product-options-wrapper select').select('XL')
+          break;
+        case 'grouped':
+          cy.get('#product_addtocart_form input[name="super_group[2]"]').clear().type('1')
+          cy.get('#product_addtocart_form input[name="super_group[3]"]').clear().type('1')
+          break;
+        case 'bundle':
+          // The pipe causes us to keep clicking until we get the stuff down below.
+          const click = $el => $el.click()
+          cy.contains('Customize and Add to Cart').should('be.visible').pipe(click).should(() => {
+            const finalButton = Cypress.$('#product-addtocart-button')
+            expect(finalButton).to.be.visible
+          })
+          break;
+        case 'simple':
+          // Do nothing
+          break;
+        default:
+          throw 'Methinks thou hast forgotten something…'
+      }
```

Comment:
I really need to set up eslint. I think this whole block is indented one too many.

Actually, looking further, I think that the JS standard is two space indents, and this file is partially indented with four spaces. Could we swap back to two spaces?